include/wx/osx/clipbrd.h

```diff
@@ -19,6 +19,101 @@
 // wxClipboard
 //-----------------------------------------------------------------------------
 
+class WXDLLIMPEXP_CORE wxOSXDataSourceItem
+{
+public:
+    virtual ~wxOSXDataSourceItem();
+
+    virtual wxDataFormat::NativeFormat AvailableType(CFArrayRef types) const = 0;
+
+    virtual bool GetData( const wxDataFormat& dataFormat, wxMemoryBuffer& target) = 0;
```

Comment:
It looks like this `GetData()` overload always will/does forward to the other one, so why make it pure virtual (or even virtual at all)?